packages/react/package.json

```diff
@@ -61,7 +61,7 @@
     "typescript": "^5.4.5"
   },
   "dependencies": {
-    "@web3-onboard/core": "2.22.2",
+    "@web3-onboard/core": "2.22.3-alpha.1",
```

Comment:
Please bump the version of this file as well to `2.9.3-alpha.1` on line 3 above so the change is published along with the latest dependencies. Please do the same for vue and solid

Reply:
This was handled incorrectly so I made some updates and pushed them